server/boot/create-default-tables.js

```diff
@@ -220,6 +220,11 @@ module.exports = function(app) {
           'description': 'Checks to see if Facebook Messenger is working',
           'spec_url': 'https://github.com/TheTorProject/ooni-spec/blob/master/test-specs/ts-019-facebook-messenger.md'
       },
+          'name': 'telegram',
```

Comment:
I should really get used to testing things before merging them...

This contains invalid markup and leads to error.

I am going to revert this commit.